community/kernel/src/main/java/org/neo4j/kernel/GraphDatabaseDependencies.java

```diff
@@ -26,14 +26,21 @@
 import org.neo4j.helpers.Service;
 import org.neo4j.helpers.collection.Iterables;
 import org.neo4j.kernel.extension.KernelExtensionFactory;
+import org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory;
 import org.neo4j.kernel.impl.query.QueryEngineProvider;
 import org.neo4j.logging.LogProvider;
 import org.neo4j.kernel.monitoring.Monitors;
 
 import static org.neo4j.helpers.collection.Iterables.addAll;
 
-public class GraphDatabaseDependencies implements InternalAbstractGraphDatabase.Dependencies
+public class GraphDatabaseDependencies implements GraphDatabaseFacadeFactory.Dependencies
 {
+    public static GraphDatabaseDependencies newDependencies(GraphDatabaseFacadeFactory.Dependencies deps)
+    {
+        return new GraphDatabaseDependencies( deps.monitors(), deps.userLogProvider(), Iterables.toList(deps.settingsClasses
+                () ), Iterables.toList(deps.kernelExtensions() ), Iterables.toList(deps.executionEngines() ) );
```

Comment:
I dislike the point at this line is broken into two, surely automatic line breaking should be able to be more context-aware. Also different style of space before/after parentheses.